docs/builders/iso.mdx

```diff
@@ -347,7 +347,19 @@ in the image's Cloud-Init settings for provisioning.
 
 - `bios` - (string) - Set the machine bios. This can be set to ovmf or seabios. The default value is seabios.
 
-- `efidisk` - (string) - Set the efidisk storage location. This need to be set if you use ovmf uefi boot
+- `efidisk` - (string) - This option is deprecated, please use `efi_config` instead (see below).
```

Comment:
Thanks a lot for your review!
Sorry, I misunderstood your last comment - I removed the `efidisk` section.

Do you create changelogs automatically, or should I add a line to `CHANGELOG.md` with this PR?

Reply:
Changelogs are generated, but we can edit them to add a note if something is breaking for example, so that's probably what will happen when we fold this in, so users are aware of the breaking change.